src/legacy-theme/ts/colors/dark.ts

```diff
@@ -1,503 +1,532 @@
 const colors = {
-  canvasDefaultTransparent: 'rgba(13,17,23,0)',
-  pageHeaderBg: '#0d1117',
+  canvasDefaultTransparent: 'var(--bgColor-transparent, var(--color-canvas-default-transparent, rgba(13,17,23,0)))',
+  pageHeaderBg: 'var(--page-header-bgColor, var(--color-page-header-bg, #0d1117))',
   marketingIcon: {
-    primary: '#79c0ff',
-    secondary: '#1f6feb',
+    primary: 'var(--color-marketing-icon-primary, #79c0ff)',
+    secondary: 'var(--color-marketing-icon-secondary, #1f6feb)',
   },
   diffBlob: {
     addition: {
-      numText: '#e6edf3',
-      fg: '#e6edf3',
-      numBg: 'rgba(63,185,80,0.3)',
-      lineBg: 'rgba(46,160,67,0.15)',
-      wordBg: 'rgba(46,160,67,0.4)',
+      numText: 'var(--diffBlob-addition-fgColor-num, var(--color-diff-blob-addition-num-text, #e6edf3))',
+      fg: 'var(--diffBlob-addition-fgColor-text, var(--color-diff-blob-addition-fg, #e6edf3))',
+      numBg: 'var(--diffBlob-addition-bgColor-num, var(--color-diff-blob-addition-num-bg, rgba(63,185,80,0.3)))',
+      lineBg: 'var(--diffBlob-addition-bgColor-line, var(--color-diff-blob-addition-line-bg, rgba(46,160,67,0.15)))',
+      wordBg: 'var(--diffBlob-addition-bgColor-word, var(--color-diff-blob-addition-word-bg, rgba(46,160,67,0.4)))',
     },
     deletion: {
-      numText: '#e6edf3',
-      fg: '#e6edf3',
-      numBg: 'rgba(248,81,73,0.3)',
-      lineBg: 'rgba(248,81,73,0.1)',
-      wordBg: 'rgba(248,81,73,0.4)',
+      numText: 'var(--diffBlob-deletion-fgColor-num, var(--color-diff-blob-deletion-num-text, #e6edf3))',
+      fg: 'var(--diffBlob-deletion-fgColor-text, var(--color-diff-blob-deletion-fg, #e6edf3))',
+      numBg: 'var(--diffBlob-deletion-bgColor-num, var(--color-diff-blob-deletion-num-bg, rgba(248,81,73,0.3)))',
+      lineBg: 'var(--diffBlob-deletion-bgColor-line, var(--color-diff-blob-deletion-line-bg, rgba(248,81,73,0.1)))',
+      wordBg: 'var(--diffBlob-deletion-bgColor-word, var(--color-diff-blob-deletion-word-bg, rgba(248,81,73,0.4)))',
     },
     hunk: {
-      numBg: 'rgba(56,139,253,0.4)',
+      numBg: 'var(--diffBlob-hunk-bgColor-num, var(--color-diff-blob-hunk-num-bg, rgba(56,139,253,0.4)))',
     },
     expander: {
-      icon: '#7d8590',
+      icon: 'var(--diffBlob-expander-iconColor, var(--color-diff-blob-expander-icon, #848d97))',
```

Comment:
Note that these changes from `7d8590` to `848d97` are actually intended and correct ✅